scripts/page_browse.js

```diff
@@ -352,7 +352,7 @@ function pageBrowse(params, storageKey, replaceUrl, mentorMode = false, setShowF
     }
 
     getProjectData = function () {
```

Comment:
Should this be renamed `getProjectTitle` since it's no longer returning the whole project data?

Reply:
That seems logical.